server.py

```diff
@@ -62,7 +62,7 @@ def do_POST(self):
     def do_PUT(self):
         """Handles PUT requests to the server
         """
-        self._set_headers(204)
+        self._set_headers(201)
```

Comment:
Recommend adding logic to evaluate the true/false returned by the sql_helper function.  Right now, nothing is setting the 404 if the PUT is unsuccessful.  Also, should this stay as a 204 No Content since a response isn't expected back from the API?